README.md

```diff
@@ -1 +1,116 @@
-# wavefront-sdk-docs
+# Wavefront SDK Documentation Sources
+
+Wavefront supports a suite of open source SDKs that developers can use to instrument applications for observability. The instrumented application collects and sends metrics, histograms, and/or trace data to Wavefront for storage and visualization.
+
+## Table of Content
+* [What's in this repository](#What's-in-this-repository)
+* [Wavefront SDKs](#Wavefront-SDKs)
+* [License](#License)
+* [How to Contribute](#How-toContribute)
+
+## What's in this repository
+Welcome to the repository that contains documentation common to the Wavefront's SDKs! 
+
+> **Note**: This repository does not include the steps on how to instrument your application using an SDK. For details on using an SDK, see the [table below](#Wavefront-SDKs) and click on a SDK under Supported Languages.
```

Comment:
**Note**: For details on our SDKs and how to use them, click one of the links in the table below.